### PR TITLE
Add mn-MN translation

### PR DESCRIPTION
**Pull request by tengis**
Hey Wojciech,

First of all, thanks for making this awesome diagram. And here is a small contribution for it. It's not perfect but I think it's good start for Mongolian translation. 

Cheers.

### CODE REVIEW

**Comment by wojtekmaj** [comment]
Thanks! Think you can find someone else speaking Mongolian to have a look if you are unsure? :)

**Comment by tengis** [comment]
Hi @wojtekmaj,

Absolutely,

hey @dulmandakh whenever you have a spare time - can you please take a look my translation here? Thank you! 🙏

File: src/i18n/json/mn-MN.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "React lifecycle methods diagram": "React амьдарлын мөчлөгийн диаграмм",
+  "Options": "Тохиргоо",
+  "Show less common lifecycles": "Түгээмэл бус мөчлөгийн функцуудыг харуул",
+  "React version": "React хувилбар",
+  "Language": "Хэл",
+  "Mounting": "DOM-д бэхлэх (Mounting)",
```
Comment (wojtekmaj) [inline src/i18n/json/mn-MN.json]: Not sure about this, it's not really mounting DOM, but the component, isn't it?

Comment (tengis) [reply src/i18n/json/mn-MN.json]: Yes. This actually hard one to translate. So meaning of "DOM-д бэхлэх" is actually saying mounting components into DOM or like attaching component into DOM tree.

Comment (tengis) [reply src/i18n/json/mn-MN.json]: so **Mounting** inside parentheses represents actual word.

Comment (wojtekmaj) [reply src/i18n/json/mn-MN.json]: It is not either, though. This is not the moment in which DOM operations are happening. Please, let's stick to the original string as closely as possible!

**Comment by wojtekmaj** [comment]
@tengis Are you willing to resolve conflicts & fix the issues we've discussed or should I close this PR?

File: src/i18n/json/mn-MN.json (hunk 1 above)
Comment (tengis) [reply src/i18n/json/mn-MN.json]: hey @wojtekmaj,

Sorry it took so long - I have updated the translation according to this doc --> https://mn.reactjs.org/docs/react-component.html#componentdidmount

So hopefully, it's now consistent and close to what it is. 😄 

Cheers

**Pull request closed without merging**